PauloAlves8039/angular-parking-frontend
# [FEATURE] Created pagination component with pipes for some fields

### Context
This Pull Request introduces the creation of the paging component and the pipes for some fields in the modal entries and data outputs in the tables of all the other components.

### Changes
  - **Pipes:** Created `DateformatPipe`, `DatehourPipe`, `RealPipe`, `CpfmaskPipe`, `ZipcodeformatPipe`
  - **All Tables Components:** Changed `StayTableComponent`, 
                                `AddressTableComponent`, 
								`CustomerTableComponent`, 
								`CustomerVehicleTableComponent`,
								`VehicleTableComponent`.

### How to Test
  1. Navigate to the all components feature in the application.
  2. Verify that the all tables is correctly populated.
  3. Test the save, update, and delete operations via the modals.
  4. Ensure that routing between components works as expected.
  
### Notes
  - Ensure all new components and services are properly tested and integrated.
  - Review the routing logic to confirm all paths are covered.

### Related Issues
  - [Link to related issues or tasks, if applicable.]
